adminapi/filters.py

```diff
@@ -78,12 +97,17 @@ def __repr__(self):
         return 'Comparison({0!r}, {1!r})'.format(self.comparator, self.value)
 
     def serialize(self):
-        return {
-            'name': 'comparison',
-            'comparator': self.comparator,
-            'value': self.value,
-        }
+        return {type(self).__name__: [self.comparator, self.value]}
 
+    @classmethod
+    def deserialize_value(cls, value):
+        if not isinstance(value, list) or len(value) != 2:
+            raise FilterValueError(
+                'Invalid value for {}()'.format(cls.__name__)
+            )
+        return cls(value[0], value[1])
+
+    # TODO Remove
```

Comment:
For other developers it could be interesting to know, why this should be removed.

You know, like you didn't get in touch with this code anymore and in 5 years another developers look at it and don't know why it should be removed.

Reply:
Also:
* 142
* 177
* 212
* 227
* 261
* 308
* 314
* 78
* 56

Reply:
This commit changes the serialization format of the filters.  Those `from_obj()` methods are for backwards compatibility.  I will create another merge request to get rid of them immediately after merging this one.